lib/src/casting/casting_config.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'casting_config.g.dart';
+
+/// Arguments to send a message
+@JsonSerializable(explicitToJson: true)
+class BitmovinCastManagerSendMessage extends Equatable {
```

Comment:
Nit: How about we go for a crisper name, e.g. `CustomCastMessage` or just `CastMessage`

Reply:
Done: 55631f45bc62942d63791d9ace89dc6afc2d4d16